keps/sig-node/3066-subsecond-probes/README.md

```diff
@@ -0,0 +1,859 @@
+high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99.9% of /health requests per day finish with 200 code
+
+These goals will help you determine what you need to measure (SLIs) in the next
+question.
+-->
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+<!--
+Pick one more of these and delete the rest.
+-->
+
+TBD
+
+- [ ] Metrics
+  - Metric name:
+  - [Optional] Aggregation method:
+  - Components exposing the metric:
+- [ ] Other (treat as last resort)
+  - Details:
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+<!--
+Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+implementation difficulties, etc.).
+-->
+### Dependencies
+
+This section will be completed when targeting beta graduation to a release.
+
+###### Does this feature depend on any specific services running in the cluster?
+
+<!--
+Think about both cluster-level services (e.g. metrics-server) as well
+as node-level agents (e.g. specific version of CRI). Focus on external or
+optional services that are needed. For example, if this feature depends on
+a cloud provider API, or upon an external software-defined storage or network
+control plane.
+
+For each of these, fill in the following—thinking about running existing user workloads
+and creating new ones, as well as about cluster-level services (e.g. DNS):
+  - [Dependency name]
+    - Usage description:
+      - Impact of its outage on the feature:
+      - Impact of its degraded performance or high-error rates on the feature:
+-->
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+Enabling / using this feature will not result in any new API calls.
+See earlier KEP text regarding limits and scaling (to zero the millisecond offsets)
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No new scalability API types.
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No new calls to the cloud provider will be provided.
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+Enabling this feature will not necessarily result in increasing size or count of
+the existing API objects. (3 optional omit if nil *int32 fields for each probe)
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+Enabling / using this feature will result in changing the time taken (and thus charging) by certain
+operations covered by [existing SLIs/SLOs]. Whether Negatively/Positively will be determined by
```

Comment:
Can you be more specific? I don't understand what you mean by that (in particular which SLI/SLO).